gst/nnstreamer/tensor_demux/gsttensordemux.c

```diff
@@ -462,7 +462,7 @@ gst_tensor_demux_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
     srcpad = gst_tensor_demux_get_tensor_pad (tensor_demux, &created, i);
 
     outbuf = gst_buffer_new ();
-    mem = gst_buffer_peek_memory (buf, i);
+    mem = gst_buffer_get_memory (buf, i);
```

Comment:
This is in a for loop. With up to 16 mem (gst-buffer's memory), you are unref-ing only one of them.